minitask/new_import_data_v2.py

```diff
@@ -0,0 +1,162 @@
+
```

Comment:
**[Required, File naming]** We should just replace the original `import_data.py` with the new implementation (this or the one above). There's no reason to keep it around.

Reply:
rename minitask to import data and replace the import_data with newest one now